IPython/terminal/pt_inputhooks/__init__.py

```diff
@@ -110,7 +111,11 @@ def set_qt_api(gui):
                 del os.environ["QT_API"]
         else:
             print(f'Unrecognized Qt version: {gui}. Should be "qt5", "qt6", or "qt".')
-            return
+            return None
+        # Import it now so we can figure out which version it is.
```

Comment:
```suggestion
            return

        # Import it now so we can figure out which version it is.
```

`return` is equivalent to `return None`.